tests/run-make/apple-slow-tls/rmake.rs

```diff
@@ -0,0 +1,28 @@
+//! Test if compilation with has-thread-local=false works, and if the output
+//! has indeed no fast TLS variables.
+
+//@ only-apple
+
+use run_make_support::run::cmd;
+use run_make_support::{cargo, rfs, rustc};
+
+fn main() {
+    let output =
+        rustc().print("target-spec-json").args(["-Z", "unstable-options"]).run().stdout_utf8();
+    let to_search = r#""has-thread-local": true"#;
+    assert!(output.contains(to_search));
+    let output = output.replace(to_search, r#""has-thread-local": false"#);
+
+    let out_path = "t.json";
+    rfs::write(out_path, output);
+
+    cargo()
+        .current_dir("tls_test")
+        .args(["b", "--target", "../t.json", "-Zbuild-std=std,core,panic_abort"])
+        .run();
+
+    let output = cmd("nm").arg("tls_test/target/t/debug/tls_test").run().stdout_utf8();
+    // If a binary has any fast TLS variables, it should also contain the symbols
+    // __tlv_bootstrap and __tlv_atexit. We don't want them.
+    assert!(!output.contains("__tlv_bootstrap"));
```

Comment:
Discussion: hm, this check is quite loose. Can the `__tlv_bootstrap` and `__tlv_atexit` symbols possibly get prepended by an *additional* `_`? You may want to use e.g. `object` and check for symbols that end with `tlv_{bootstrap,atexit}` instead.

Reply:
I looked at the `object` crate, but it seemed a bit scary to use, so I went with `llvm_nm`, and searching for symbols with just one prefix underscore.